exercises/flatten-array/package.json

```diff
@@ -48,23 +53,71 @@
     },
     "extends": "eslint:recommended",
     "rules": {
-      "indent": [ "off", 2 ],
+      "indent": [
+        "off",
+        2
+      ],
       "block-scoped-var": "off",
       "radix": "off",
       "no-use-before-define": "off",
-      "one-var": [ "off", "always" ],
-      "quotes": [ "off", "single", { "avoidEscape": true } ],
-      "semi": [ "off", "always" ],
-      "semi-spacing": [ "off", { "before": false, "after": true } ],
+      "one-var": [
+        "off",
+        "always"
+      ],
+      "quotes": [
+        "off",
+        "single",
+        {
+          "avoidEscape": true
+        }
+      ],
+      "semi": [
+        "off",
+        "always"
+      ],
+      "semi-spacing": [
+        "off",
+        {
+          "before": false,
+          "after": true
+        }
+      ],
       "no-whitespace-before-property": "off",
-      "space-before-blocks": [ "off", "always" ],
-      "space-before-function-paren": [ "off", { "anonymous": "always", "named": "never" } ],
-      "keyword-spacing": [ "off", { "before": true, "after": true } ],
+      "space-before-blocks": [
+        "off",
+        "always"
+      ],
+      "space-before-function-paren": [
+        "off",
+        {
+          "anonymous": "always",
+          "named": "never"
+        }
+      ],
+      "keyword-spacing": [
+        "off",
+        {
+          "before": true,
+          "after": true
+        }
+      ],
       "no-multi-spaces": "off",
       "no-trailing-spaces": "off",
-      "curly": [ "error", "all" ],
-      "brace-style": [ "error", "1tbs", { "allowSingleLine": true } ],
-      "object-curly-spacing": [ "off", "always" ]
+      "curly": [
+        "error",
+        "all"
+      ],
+      "brace-style": [
+        "error",
+        "1tbs",
+        {
+          "allowSingleLine": true
+        }
+      ],
+      "object-curly-spacing": [
+        "off",
+        "always"
```

Comment:
It looks like the formatting of these three `package.json` files was out-of sync with `exercises/package.json`.  This change brings them in line.  

I didn't intentionally add it, (Makefile copied it over) but I think it needed to be done and can remain here rather than making a separate PR.